src/System.Diagnostics.Process/System.Diagnostics.Process.sln

```diff
@@ -24,8 +24,8 @@ Global
 		Windows_Release|Any CPU = Windows_Release|Any CPU
 	EndGlobalSection
 	GlobalSection(ProjectConfigurationPlatforms) = postSolution
-		{63634289-90D7-4947-8BF3-DBBE98D76C85}.Linux_Debug|Any CPU.ActiveCfg = Linux_Debug|Any CPU
-		{63634289-90D7-4947-8BF3-DBBE98D76C85}.Linux_Debug|Any CPU.Build.0 = Linux_Debug|Any CPU
+		{63634289-90D7-4947-8BF3-DBBE98D76C85}.Linux_Debug|Any CPU.ActiveCfg = Windows_Release|Any CPU
+		{63634289-90D7-4947-8BF3-DBBE98D76C85}.Linux_Debug|Any CPU.Build.0 = Windows_Release|Any CPU
```

Comment:
This change doesn't do that, though... it's changing the project used for the "Linux_Debug" configuration to be "Windows_Release", which doesn't make sense :)